spec/support/shared_examples/phone/rate_limitting.rb

```diff
@@ -42,11 +42,16 @@
 
   it 'limits the number of times the user can enter an OTP' do
     visit_otp_confirmation(delivery_method)
-    (max_confirmation_attempts - 1).times do
+    (max_confirmation_attempts - 1).times do |number_of_times|
       fill_in :code, with: '123456'
       click_submit_default
-
-      expect(page).to have_content(t('two_factor_authentication.invalid_otp'))
+      current_count = max_confirmation_attempts - number_of_times
+      expect(page).to have_content(
+        t(
+          'two_factor_authentication.invalid_otp',
+          count: current_count,
```

Comment:
The `invalid_otp` string no longer receives the `count` variable. Should we update this spec? I think it'd be good to check both scenarios where the count isn't (yet) shown and then when it is.